core/src/toga/app.py

```diff
@@ -106,7 +106,7 @@ def __iter__(self) -> Iterator[Widget]:
         return self.values()
 
     def __repr__(self) -> str:
-        return f"{{{', '.join(f'{k!r}: {v!r}' for k, v in sorted(self._registry.items()))}}}"
+        return f"{{{', '.join(f'{k!r}: {v!r}' for k, v in sorted(self._registry.items()))}}}"  # noqa: E501
```

Comment:
This line can be split as shown in https://github.com/beeware/toga/pull/2976/commits/76548204b4ca756ce350b74b8369fb400bc30079.

Reply:
Changed